NextcloudTalk/NCChatMessage.m

```diff
@@ -141,11 +168,12 @@ - (BOOL)isMessageFromUser:(NSString *)userId
     return [self.actorId isEqualToString:userId] && [self.actorType isEqualToString:@"users"];
 }
 
-- (BOOL)isDeletableForUserId:(NSString *)userId andParticipantType:(NCParticipantType)participantType
+- (BOOL)isDeletableForAccount:(TalkAccount *)account andParticipantType:(NCParticipantType)participantType
 {
     NSInteger sixHoursAgoTimestamp = [[NSDate date] timeIntervalSince1970] - (6 * 3600);
-    if ([self.messageType isEqualToString:kMessageTypeComment] && !self.file && self.timestamp >= sixHoursAgoTimestamp &&
-        (participantType == kNCParticipantTypeOwner || participantType == kNCParticipantTypeModerator || [self isMessageFromUser:userId])) {
+    BOOL canServerDeleteMessages = [[NCSettingsController sharedInstance] serverHasTalkCapability:kCapabilityChatReadStatus forAccountId:account.accountId];
```

Comment:
Really Read-Status here ?

Reply:
😬🙈 Thanks! 😉

Reply:
Fixed in c8623f8a33d1f853b08b2b7dd926a557a22396ca

Reply:
👍